lib/packages/libbsdgames2.rb

```diff
@@ -0,0 +1,85 @@
+    system "for i in \$(cat debian/patches/series); do patch -Np1 -i debian/patches/\"\${i}\"; done"
+    system "sed -i 's:-lncurses:-lncurses -ltinfo:g' configure" # Add libtinfo support
+    system "sed -i 's:-lcrypto:-lssl -lcrypto:g' configure" # Fix openssl
+    system "sed -i 's:#!/bin/sh:#!/bin/bash:' configure" # dash doesn't work
+    system "sed -i 's:read input:read -t 0.2 input:' configure" # Make script automatic
+  end
+
+  def self.prebuild
+    puts 'Configuring'.lightblue
+    system "sed -i 's:-g -O2:#{CREW_COMMON_FLAGS}:g' configure"
+    system "sed -i 's:/usr/games:#{CREW_PREFIX}/bin:' wargames/wargames"
+    system "echo bsd_games_cfg_build_dirs='\"#{@bsdgame}\"' >> config.params"
+    system "echo bsd_games_cfg_docdir='#{CREW_PREFIX}/share/doc/bsdgames' >> config.params"
+    system "echo bsd_games_cfg_install_prefix='#{CREW_DEST_DIR}' >> config.params"
+    system "echo bsd_games_cfg_gamesdir='#{CREW_PREFIX}/bin' >> config.params"
+    system "echo bsd_games_cfg_sbindir='#{CREW_PREFIX}/sbin' >> config.params"
+    system "echo bsd_games_cfg_usrbindir='#{CREW_PREFIX}/bin' >> config.params"
+    system "echo bsd_games_cfg_libexecdir='#{CREW_PREFIX}/libexec' >> config.params"
+    system "echo bsd_games_cfg_man6dir='#{CREW_MAN_PREFIX}/man6' >> config.params"
+    system "echo bsd_games_cfg_man8dir='#{CREW_MAN_PREFIX}/man8' >> config.params"
+    system "echo bsd_games_cfg_man5dir='#{CREW_MAN_PREFIX}/man5' >> config.params"
+    system "echo bsd_games_cfg_do_chown='n' >> config.params" # Use chronos not root
+    system "echo bsd_games_cfg_use_dot_so='symlinks' >> config.params"
+    system "echo bsd_games_cfg_sharedir='#{CREW_PREFIX}/share/bsdgames' >> config.params"
+    system "echo bsd_games_cfg_varlibdir='#{CREW_PREFIX}/var/bsdgames' >> config.params"
+    system "echo bsd_games_cfg_ncurses_includes='-I#{CREW_PREFIX}/include/ncursesw' >> config.params"
+    system "echo bsd_games_cfg_use_libcrypto='n' >> config.params"
+    system "echo bsd_games_cfg_other_cflags='\"#{CREW_COMMON_FLAGS}\"' >> config.params" # Link-time optimization
+    system "echo bsd_games_cfg_other_ldflags='\"#{CREW_LDFLAGS}\"' >> config.params"
+    system "echo bsd_games_cfg_pager='$(which less)' >> config.params" # Most looks weird and more doesn't scroll up
+    system "echo bsd_games_cfg_dictionary_src='#{CREW_PREFIX}/share/dict/words' >> config.params"
+    system "echo bsd_games_cfg_dm_configfile='#{CREW_PREFIX}/etc/dm/dm.conf' >> config.params"
+    system "echo bsd_games_cfg_dm_nogamesfile='#{CREW_PREFIX}/etc/dm/nogames' >> config.params"
+    system "echo bsd_games_cfg_hangman_wordsfile='#{CREW_PREFIX}/share/dict/words' >> config.params"
+    system "echo bsd_games_cfg_wtf_acronymfile='#{CREW_PREFIX}/share/misc/acryonyms' >> config.params"
+    system "echo bsd_games_cfg_ncurses_lib='$(pkg-config --libs-only-l ncurses)' >> config.params" # Fix ncurses support
+  end
+
+  def self.build
+    system "./configure"
```

Comment:
Shouldn't we be using `#{CREW_OPTIONS}`?

Reply:
This isn't an autotools configure script, it was written custom for bsdgames. #{CREW_OPTIONS} won't work with it.